modules/assim.sequential/R/sda_plotting.R

```diff
@@ -539,8 +539,11 @@ post.analysis.multisite.ggplot <- function(settings, t, obs.times, obs.mean, obs
               Lower=quantile(Value,0.025, na.rm=T),
               Upper=quantile(Value,0.975, na.rm=T))
           
-        })%>%mutate(Type=paste0("SDA_",listFA),
+        # dropped the "_" from "SDA_" in plot legends
```

Comment:
I guess @mdietze meant the whole "SDA_" and not just "_" and as a heads up, when you take the "SDA" out this would change the order of colors on the plots.

Reply:
oh lol. that's an easy fix